wpeverest/everest-forms
# Enhancement - Pass Field Data to Query String.

### All Submissions:


### Changes proposed in this Pull Request:
Previously, there was no method to send the field field data to query string with custom page redirect.

### How to test the changes in this Pull Request:

1. Go to the general Setting of Form Builder.
2. Choose the custom page in Submission redirection and enable the redirect to query string.
3. Write query string like description given there and verify it is working or not

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Pass Field Data to Query String..
